phalcon/Events/AbstractEventsAware.zep

```diff
@@ -13,11 +13,9 @@ namespace Phalcon\Events;
 use Phalcon\Events\ManagerInterface;
 
 /**
- * Trait EventsAwareTrait
+ * Phalcon\Events\AbstractEventsAware
```

Comment:
I would prefer we do not add this. It makes searching for classes/names harder. The `Trait EventsAwareTrait` can also go.